lib/src/chart/pie_chart/pie_chart_data.dart

```diff
@@ -401,3 +414,9 @@ class PieChartDataTween extends Tween<PieChartData> {
   @override
   PieChartData lerp(double t) => begin!.lerp(begin!, end!, t);
 }
+
+enum CapStyle {
```

Comment:
Please write a documentation above it. And it seems it is specially for PieChart. So I think it makes sense to add `Pie` prefix for it.